### PR TITLE
#5171 uiSchema in integration definition not taking effect in integration configuration

### LINKED ISSUE #5171: uiSchema in integration definition not taking effect in integration configuration
Context
---
- We had added support for using uiSchema in integration and brick configuration to be able to specify the input parameter entry order

Screenshot
--- 
image

image

### PR DESCRIPTION
**Pull request by BALEHOK**
## What does this PR do?

- Fixes #5171 

## Checklist

- [ ] Add tests
- [x] Designate a primary reviewer @BLoe

### CODE REVIEW

**Comment by twschiller** [comment]
@BLoe could you pick this one up? I think the change is likely correct, but let's also add a test here that it sets uiSchema: https://github.com/pixiebrix/pixiebrix-extension/blob/bfb1eefd9bde6c4f1fd98598ab5dccaa517e68f4/src/services/factory.test.ts#L26-L26

**Comment by twschiller** [comment]
@BLoe could you pick this one up? I think the change is likely correct, but let's also add a test here that it sets uiSchema: https://github.com/pixiebrix/pixiebrix-extension/blob/bfb1eefd9bde6c4f1fd98598ab5dccaa517e68f4/src/services/factory.test.ts#L26-L26

**Comment by twschiller** [comment]
@BLoe could you pick this one up? I think the change is likely correct, but let's also add a test here that it sets uiSchema: https://github.com/pixiebrix/pixiebrix-extension/blob/bfb1eefd9bde6c4f1fd98598ab5dccaa517e68f4/src/services/factory.test.ts#L26-L26

**Comment by twschiller** [comment]
@BLoe could you pick this one up? I think the change is likely correct, but let's also add a test here that it sets uiSchema: https://github.com/pixiebrix/pixiebrix-extension/blob/bfb1eefd9bde6c4f1fd98598ab5dccaa517e68f4/src/services/factory.test.ts#L26-L26

**Comment by BLoe** [comment]
@twschiller Added `uiSchema` to the test here:
https://github.com/pixiebrix/pixiebrix-extension/pull/5208/files#diff-82e5a89e875ea787416414a1cdd0574ad2537b5a5c2a87aacf67a90e41954bceR27-R33

File: src/services/factory.test.ts [hunk 1]
```diff
@@ -27,6 +27,9 @@ describe("LocalDefinedService", () => {
   test("includes version", () => {
     const service = fromJS(automationAnywhere as unknown as ServiceDefinition);
     expect(service.version).toBe("1.0.0");
+    expect(service.uiSchema).toMatchObject({
```
Comment (twschiller) [inline src/services/factory.test.ts:30]: 👍

**Review by twschiller — APPROVED** [review]

**Pull request merged**